marathon_acme/tests/test_clients.py

```diff
@@ -665,6 +666,9 @@ def test_get_events(self):
         request.finish()
         yield d
 
+        # Expect request.finish() to result in a logged failure
+        flush_logged_errors(ResponseDone)
```

Comment:
The testtools documentation says _"Use this function to declare that logged errors were expected behavior."_ But as far as I can tell it won't assert that the error was flushed. We probably want at least one test to check that but I'm not sure how at the moment.